Sources/TuistKit/Commands/ImportFinderCommand.swift

```diff
@@ -0,0 +1,39 @@
+import AnyCodable
+import ArgumentParser
+import Foundation
+import Path
+import TuistSupport
+
+public struct ImportFinderCommand: AsyncParsableCommand {
```

Comment:
I'd suggest removing this one as a command that users directly interface with. I think that command should be `tuist lint implicit-imports` instead, which can be implemented in a follow-up PR.

Reply:
I agree with you, but now I need a way to test it. So I remove this command, when we will be ready to merge

Reply:
Sounds good! Thanks for clarifying @rofle100lvl